lib/components/src/icon-tabs/icon-tab.ts

```diff
@@ -0,0 +1,27 @@
+import { customElement, property } from 'lit/decorators.js';
+import { TabBase } from '@material/mwc-tab/mwc-tab-base';
+import {styles as tabBarStyles} from '@material/mwc-tab/mwc-tab.css';
+import styles from "./icon-tab.scss";
+
+declare global {
+  interface HTMLElementTagNameMap {
+   'td-icon-tab': CovalentIconTabBase;
+  }
+}
+
+@customElement('td-icon-tab')
+export class CovalentIconTabBase extends TabBase {
+  static override styles = [styles, tabBarStyles];
+  @property({type: String}) secondLine = 'Label';
```

Comment:
should this variable be blank by default?

Reply:
Yeah it should be